API-strategie-modulen/API-strategie-mod-geo/request-response.md

```diff
@@ -1,7 +1,15 @@
 # Request and response
-There are several challenges related to handling geospatial features in REST APIs. There is a distinction between retrieving geometries in the result (response) and supplying a geometry in the call (request). 
 
-Note that when requesting information, for example about cadastral parcels, users do not necessarily require the geometry. A name or parcel ID may be sufficient. Geometries can also be part of the request, for example as a spatial filter or when creating or updating a spatial feature. When geometry is part of the request or response, several rules apply, which are described below. 
+Providing requested resources is the essence of any API. This also applies to REST APIs that handle geospatial data. There are, however, some specific aspects when dealing with geospatial data in REST APIs. The most important aspects are described in this chapter: 
+- how to encode geometries in APIs
```

Comment:
Gaat dit over request en response of alleen over de request?

Reply:
Over beide.